include/ck/tensor_description/merge_transform_for_wrw.hpp

```diff
@@ -0,0 +1,161 @@
+#pragma once
+
+#include "common_header.hpp"
+#include "multi_index_transform.hpp"
+
+namespace ck {
+
+// Implementation of "Merge" transformation primitive that uses division and mod. It is supposed to
+// be used for low_lengths that are known at compile time and are power of 2, otherwise performance
+// will be very bad
+template <typename LowLengths>
+struct Merge_v3_division_mod_for_wrw
```

Comment:
Since this ```Merge_v3_division_mod_for_wrw``` is used by ```DeviceConv2dBwdWeightXdl_C_Shuffle_Input_N_Hi_Wi_C_Weight_K_Y_X_C_Output_N_Ho_Wo_K``` only, please move the definition of former into the same file of latter

Also I think a name like```Merge_v4_no_carry``` maybe a more meaningful name. Also please add comment to describe what it does.

Reply:
Yes, I agree.

Reply:
Moved and renamed.